prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -250,11 +251,40 @@ class CompareCsvFiles : CliktCommand(
     }
 }
 
+class LivdTableDownload() : CliktCommand(
+    name = "livd-table",
+    help = """
+    livd-table downloads the latest LOINC test data, so it can be ingested automatically. 
+    
+    It looks for the LIVD-SAR-CoV-2-yyyy-MM-dd.xlsx file from https://www.cdc.gov/csels/dls/sars-cov-2-livd-codes.html.
```

Comment:
Try not to hard code this URL.  It is already defined in the other class, so use it.

Reply:
Done.  Thanks.